app/pktgen-port-cfg.c

```diff
@@ -234,6 +234,8 @@ pktgen_config_ports(void)
             pinfo->lsc_enabled = 1;
         }
 
+	conf.rxmode.offloads &= pinfo->dev_info.rx_offload_capa;
```

Comment:
This code change has a formatting problem and does not pass the clang-format test.

Please copy the .githooks/pre-commit file into .git/hooks directory or run clang-format, which you may need to install on your system. Using the pre-commit hook should fix the problem. Maybe best to run clang-format by hand on the file and commit/push the changes.

`clang-format -style=file -i app/pktgen-port-cfg.c; git add app/pktgen-port-cfg.c;`

Should fix the formatting problem. Then you need to commit/push the new change.

Reply:
Thanks for the cmd.